doc/pic/hld_fpmsyncd.md

```diff
@@ -484,6 +564,7 @@ Mention whether this feature/enhancement has got any requirements/dependencies/i
 
 - When the feature is disabled, there should be no impact to Warmboot and Fastboot.
 - When the feature is enabled, there will be no warmboot nor fastboot support.
```

Comment:
Fast boot can be supported. Only WB will have limitations due to reconciliation of nhids.

Reply:
Done